document/js-api/index.bs

```diff
@@ -1349,21 +1349,26 @@ In practice, an implementation may run out of resources for valid modules below
 <ul>
 <li>The maximum size of a module is 1,073,741,824 bytes (1 GiB).</li>
 <li>The maximum number of types defined in the types section is 1,000,000.</li>
+<li>The maximum number of recursion groups defined in the types sections is 1,000,000.</li>
+<li>The maximum number of types defined in a recursion group is 1,000,000.</li>
+<li>The maximum depth of a defined subtype hierarchy is 63 (where a type defined with no supertype has depth 0).
 <li>The maximum number of functions defined in a module is 1,000,000.</li>
 <li>The maximum number of imports declared in a module is 100,000.</li>
 <li>The maximum number of exports declared in a module is 100,000.</li>
 <li>The maximum number of globals defined in a module is 1,000,000.</li>
 <li>The maximum number of data segments defined in a module is 100,000.</li>
 
 <li>The maximum number of tables, including declared or imported tables, is 100,000.</li>
-<li>The maximum size of a table is 10,000,000.</li>
+<li>The maximum size of a table is 10000000.</li>
```

Comment:
Was this line a typo or merge issue maybe? (because it removes the line separators) @tlively

Reply:
Oops, thanks for the catch.